src/boot/bootctl-systemd-efi-options.c

```diff
@@ -1,12 +1,16 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
 #include "alloc-util.h"
+#include "bootctl.h"
 #include "bootctl-systemd-efi-options.h"
```

Comment:
Not sorted if we use `C.utf-8`, (but _is_ sorted, if we use `en_US.utf-8`).

Reply:
Does it really matter? I think logically "bootctl.h" should be before "bootctl-systemd-efi-options.h".